gk/main.c

```diff
@@ -660,6 +658,46 @@ add_ggu_policy(struct ggu_policy *policy,
 }
 
 static void
+gk_flush_flow_table(struct gk_fib *fib, struct gk_instance *instance)
+{
+	uint32_t next = 0;
+	int32_t index;
+	const struct ip_flow *key;
+	void *data;
+
+	index = rte_hash_iterate(instance->ip_flow_hash_table,
+		(void *)&key, &data, &next);
```

Comment:
Actually, it's necessary. Otherwise, it will pop an ``incompatible-pointer-types`` error.